### PR TITLE
15/feat send on enter and improved example

### LINKED ISSUE #15: Allow send message on enter

### PR DESCRIPTION
**Pull request by spaenleh**
This PR improves the example project and adds the ability to send messages when enter is pressed. 
Users can also use shift+enter to add a new line in the input field.

Also added a `.env.local` with the following content:
```
REACT_APP_API_HOST="http://localhost:3000"
```
This lets the example project use the local backend

closes #15

### CODE REVIEW

**Comment by spaenleh** [comment]
Added multiline display of comments
Screenshot 2022-03-04 at 11 41 04

File: example/package.json [hunk 1]
```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start",
+    "start": "env-cmd -f .env.local react-scripts start",
```
Comment (pyphilia) [inline example/package.json:7]: Good, maybe let's add in the readme that one needs to create this file. Should this file go in the root of the project?

File: example/src/components/ChatboxWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    id: 'a1112eb7-7f28-4fdb-92cc-28171177463f',
+  });
+  // todo: fetch other members using hook
+  const members = [
+    member,
+    {
+      name: 'bob',
+      id: 'bob-id',
+    },
+  ];
```
Comment (pyphilia) [inline example/src/components/ChatboxWrapper.tsx:24]: Maybe let's factor out in a `fixtures` file. Use the member mutation if you think it fits, I don't mind having a mock member if it works.

File: example/src/config/constants.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+export const ACTION_CELL_WIDTH = 230;
+export const THUMBNAIL_ASPECT = 1;
+export const THUMBNAIL_EXTENSION = 'image/jpeg';
+export const THUMBNAIL_SETTING_MAX_WIDTH = 300;
+export const THUMBNAIL_SETTING_MAX_HEIGHT = 200;
+
+export const CATEGORY_TYPES = {
+  LEVEL: 'level',
+  DISCIPLINE: 'discipline',
+};
```
Comment (pyphilia) [inline example/src/config/constants.js:164]: Can you clean this file and only keep what you need?

File: example/src/config/i18n.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  // tr: "Türkçe",
+  // uk: "Українська",
+  // vi: "Tiếng Việt",
+  // zh: "简体中文",
+  // zh_tw: "繁體中文",
+};
+
+export { langs };
+
+export default i18n;
```
Comment (pyphilia) [inline example/src/config/i18n.js:61]: I'm surprised you need translations, do you need this file?

File: example/src/env.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "API_HOST": null,
+  "SHOW_NOTIFICATIONS": false,
+  "AUTHENTICATION_HOST": false,
+  "NODE_ENV": false,
+  "GRAASP_PERFORM_HOST": false,
+  "GA_MEASUREMENT_ID": false,
+  "HIDDEN_ITEM_TAG_ID": false
+}
```
Comment (pyphilia) [inline example/src/env.json:9]: Clean to keep only necessary variables.

File: example/src/langs/fr.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (pyphilia) [inline example/src/langs/fr.json:1]: Remove this file?

File: example/src/langs/en.json [hunk 7]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (pyphilia) [inline example/src/langs/en.json:1]: Remove this file?

File: src/components/Message.tsx [hunk 8]
```diff
@@ -52,7 +52,7 @@ const Message: FC<Props> = ({ message, currentMember, member }) => {
       {!isOwnMessage && (
         <Typography variant="subtitle2">{`${creatorName}`}</Typography>
       )}
-      <Typography variant="body1">{message.body}</Typography>
+      {message.body.split('\n').map((line) => <Typography variant="body1">{line}<br /></Typography>)}
```
Comment (pyphilia) [inline src/components/Message.tsx:55]: What about this? https://stackoverflow.com/questions/39325414/line-break-in-html-with-n

**Review by pyphilia — CHANGES_REQUESTED** [review]
Nice PR! I think it needs a bit of cleaning but it's good to go ⭐

File: example/src/config/i18n.js (hunk 4 above)
Comment (spaenleh) [reply example/src/config/i18n.js:61]: No, it was just to add it for the future, but i will remove it and we can add it when we need

File: example/src/components/ChatboxWrapper.tsx (hunk 2 above)
Comment (spaenleh) [reply example/src/components/ChatboxWrapper.tsx:28]: For this i tried using the `useMembers` hook but i has problems with `crypto` package. I think for the moment i will leave it like this but the idea is to have the example project work with the local backend and the tests work with the fixtures.

**Review by pyphilia — APPROVED** [review]

**Pull request merged**